src/index.js

```diff
@@ -1,130 +1,240 @@
-var gulp;
-var open = require( 'open' ); //jshint ignore : line
-var processhost = require( 'processhost' )();
-var when = require( 'when' );
-var _ = require( 'lodash' );
-var path = require( 'path' );
-var mocha = require( 'gulp-spawn-mocha' );
-var cmdPostfix = process.platform === 'win32' ? '.cmd' : '';
-var defaultSources = [ './src/**/*.js', './resource/**/*.js' ];
-var defaultSpecPath = [ './spec' ];
-var defaultSpecs = [ '**/*.spec.js' ];
-var defaultWatchPaths = [ './src/**/*', './spec/**/*', './resource/**/*' ];
-var processesDefined = false;
-
-process.on( 'uncaughtError', console.log );
-
-function cover( specPath, show ) {
-	var args = Array.prototype.slice.call( arguments, 2 ) || [];
-	args.unshift( specPath, true );
-	return function( /* cb */ ) {
-		return runSpecs.apply( undefined, args )
-			.on( 'end', function() {
-				if ( show ) {
-					open( './coverage/lcov-report/index.html' );
-					process.exit();
-				}
-			} );
-	};
-}
+var _ = require( "lodash" );
+var when = require( "when" );
+var open = require( "open" );
+var path = require( "path" );
+var mocha = require( "gulp-spawn-mocha" );
+var processhost = require( "processhost" )();
+var cmdPostfix = process.platform === "win32" ? ".cmd" : "";
+var defaults = {
+	allSpecPaths: [ "./spec" ],
+	behaviorSpecPaths: [ "./spec/behavior" ],
+	integrationSpecPaths: [ "./spec/integration" ],
+	specs: [ "**/*.spec.js" ],
+	watchPaths: [ "./src/**/*", "./spec/**/*", "./resource/**/*" ],
+	sourcePaths: [ "./src", "./resource" ],
+	sources: [ "**/*.js" ],
+	coverageHtml: "./coverage/lcov-report/index.html",
+	exclude: [ "!node_modules*", "!coverage*" ],
```

Comment:
Found that these need to be `!node_modules/**` and `!coverage/**` otherwise the files are still included (jscs was excluding them still from .jscsrc, but jshint would not). When updated, should update it in README too.